modules/Microsoft.Compute/diskEncryptionSets/.test/common/dependencies.bicep

```diff
@@ -16,7 +16,7 @@ resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' = {
             name: 'standard'
         }
         tenantId: tenant().tenantId
-        enablePurgeProtection: null
+        enablePurgeProtection: true
```

Comment:
Does it have to be true? If not I strongly recommend to set it to `false`. Not that we don't generate a new name each time we run. As you ran a test case there's now already a fair chance we cannot re-run the test for the next couple days :D

Reply:
Hmmm.. it was complaining it needed to be true unfortunately.. given it was working before, maybe the new API mandates this.. but I was blocked by the purge protection on KV must be set to true

Reply:
That's alright (might be new) - but then we must adopt the KeyVault deployment approach we use for Purge Protected Key Vaults like in the Batch Module's Encryption test

Reply:
No problem. Aligned it to the batch module for baseTime usage.